profiles/management/commands/retire_users.py

```diff
@@ -0,0 +1,61 @@
+"""
+Validate course prices
+"""
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand, CommandError
+from social_django.models import UserSocialAuth
+
+from dashboard.models import ProgramEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Validate prices and financial aid
+    """
+    help = "Retire one or multiple users"
```

Comment:
Can we add a usage example to the help message?

Reply:
Done @pdpinch